pandas/core/apply.py

```diff
@@ -130,8 +130,8 @@ def index(self) -> "Index":
         return self.obj.index
 
     @cache_readonly
-    def values(self):
-        return self.obj.values
+    def values(self) -> np.ndarray:
+        return self.obj._values
```

Comment:
`obj` is a DataFrame here, so is there a good reason to change `values` to `_values` ? (in practice, it's actually `_values` that is implemented in terms of `values`) 

The reason to do this for Series (`_values` being the "internal" values) doesn't really apply for DataFrame?